app/views/pages/shecoded.html.erb

```diff
@@ -0,0 +1,289 @@
+<!DOCTYPE html>
```

Comment:
This file was previously removed. I assume it came back because of the rebase.

Reply:
Oops I thought I had removed it, done.